client/readme.md

```diff
@@ -0,0 +1,37 @@
+# Elm Frontend Readme
+
+The Lunar Rocks frontend is an Elm 0.19 application.
+
+==================================================
+
+## Setup
+
+Install npm packages from the `client` directory.
+```
+npm install
+```
+
+Install [Elm](https://guide.elm-lang.org/install.html) and [elm-Live](https://github.com/wking-io/elm-live), and [uglify-js](https://www.npmjs.com/package/uglify-js).
+
+## Develop 
```

Comment:
The distinction between 'Develop' and 'Build' is unclear. A brief description of each might help this readme to be clearer.